javascript/index.js

```diff
@@ -14,15 +14,17 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  printMinutes()
+  printSeconds()
 }
 
 function printMinutes() {
-  // ... your code goes here
+  let mins= chronometer.getMinutes()
```

Comment:
mins is not the correct variable where the DOM element are. The variables are minDec and minUni (lines 8 and 9)
Also, getMinutes will throw you a number, you want a string instead and you get that by calling the twoDigitNumber